### ISSUE TITLE
AdamWeightDecayOptimizer not working when fine-tune BERT.

### ISSUE THREAD

**Issue by guotong1988** [google-research/bert#1151]
Loss and accuracy do not change and keep low.

this code is not ok:

```
self.global_step = tf.Variable(0, name="global_step", trainable=False, dtype=tf.int64)
self.train_op = create_optimizer(self.loss,0.00001,num_train_steps=20000,global_step=self.global_step,num_warmup_steps=320)
```

this code is ok:

```
self.train_op = tf.train.AdamOptimizer(0.00001).minimize(self.loss, global_step=self.global_step)
```

**Comment by guotong1988** [comment]
comment these lines:
```
#  learning_rate = tf.train.polynomial_decay(
#      learning_rate,
#      global_step,
#      num_train_steps,
#      end_learning_rate=0.0,
#      power=1.0,
#      cycle=False)
```
